### ISSUE TITLE
how to build & run

### ISSUE THREAD

**Issue by mabDc** [hetu-script/script-widget#1]
it looks like requiring a specified version of flutter to build & run?

**Comment by hythl0day** [comment]
This script widget only tested on Flutter version 2.0.3 and only on the Windows platform (desktop support). I haven't given it a try on a newer version of flutter. However, it should be easy to upgrade.

And yes it has the same goal as hydrosdk, however, libraries like this always have trouble binding and importing widgets from Flutter, many libs like this heavily rely on the author's maintenance of a separate widget lib. The goal of Hetu is to make everything simple and automatic. Although it did sacrifice the language capability.

**Comment by hythl0day** [comment]
And please communicate in English here.

**Comment by mabDc** [comment]
`gitter` may be better than qq

**Comment by rockingdice** [comment]
I've tested with flutter 2.0.1 and the latest version on stable channel without any issue.
Also tested on `MacOS`, `iOS` and `Android`. It should theoritically work on any platforms that flutter supports.
The language itself is just like other pure dart packages, so it's not picky what the version of flutter you use (But should support `extension` which is available from `flutter SDK 2.7.0` if you are planning to generate binding codes).
The autobinding codes DO bound the version you generate from. It will generate a projection to the flutter & dart libraries, so it's kind of sensitive to the api changes. 
You may want to regenerate them if you're planing to upgrade flutter to a new version.

**Comment by hythl0day** [comment]
I'll try to upgrade the script widget to latest version of Flutter and Hetu later today.

**Comment by mabDc** [comment]
build latest code  is okay, but file `example/ht-lib/main.ht` need update for running error


```
HTError (Hetu error:
[HTErrorType.parser}]
[File: ht-lib/main.ht]
[Line: 21, Column: 16]
Unexpected identifier [[]
Call stack:
```

https://github.com/hetu-script/script-widget/blob/23dac8d3a1abc6e66596369c0c57abe0f1830c58/example/ht-lib/main.ht#L21-L25

**Comment by hythl0day** [comment]
> build latest code is okay, but file `example/ht-lib/main.ht` need update for running error
> 
> ```
> HTError (Hetu error:
> [HTErrorType.parser}]
> [File: ht-lib/main.ht]
> [Line: 21, Column: 16]
> Unexpected identifier [[]
> Call stack:
> ```
> 
> and line 21 is
> 
> ```
> onTap: fun[ValueChangedInt](index) {
> ```

Yes that's a bug in the compiler of hetu. I have updated hetu package ( the latest is 0.1.1+2 ) to fix this bug. However I cannot test it till this evening.

**Comment by hythl0day** [comment]
Okay I have upgraded this widget on latest build of hetu and successfully build and run.
For now, I committed a version that use local package.
I'll update the pub.dev package later.
